error: POST en /register - Error al registrar al usuario: MongoServerError: E11000 duplicate key error collection: test.users index: email_1 dup key: { email: "alzu910@gmail.com" } - at 14/3/2024 - 19:38:17
error: POST en /register - Error al registrar al usuario: MongoServerError: E11000 duplicate key error collection: test.users index: email_1 dup key: { email: "alzu910@gmail.com" } - at 14/3/2024 - 19:41:26
error: Error al crear el producto: E11000 duplicate key error collection: test.products index: code_1 dup key: { code: "12345" } {"code":11000,"index":0,"keyPattern":{"code":1},"keyValue":{"code":"12345"},"stack":"MongoServerError: E11000 duplicate key error collection: test.products index: code_1 dup key: { code: \"12345\" }\n    at InsertOneOperation.execute (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongodb\\lib\\operations\\insert.js:48:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async executeOperationAsync (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongodb\\lib\\operations\\execute_operation.js:106:16)"}
error: Error al crear el carrito: Cast to ObjectId failed for value "657dea055049e71f39f3ae3" (type string) at path "_id" for model "products" {"kind":"ObjectId","path":"_id","reason":{},"stack":"CastError: Cast to ObjectId failed for value \"657dea055049e71f39f3ae3\" (type string) at path \"_id\" for model \"products\"\n    at SchemaObjectId.cast (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\schema\\objectId.js:250:11)\n    at SchemaType.applySetters (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\schemaType.js:1219:12)\n    at SchemaType.castForQuery (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\schemaType.js:1633:15)\n    at cast (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\cast.js:375:32)\n    at Query.cast (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\query.js:4768:12)\n    at Query._castConditions (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\query.js:2200:10)\n    at model.Query._findOne (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\query.js:2484:8)\n    at model.Query.exec (C:\\Users\\lucia\\Desktop\\Backend-main\\node_modules\\mongoose\\lib\\query.js:4290:80)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ProductServiceMongo.findById (file:///C:/Users/lucia/Desktop/Backend-main/src/services/dao/mongo/product.service.js:53:20)","stringValue":"\"657dea055049e71f39f3ae3\"","value":"657dea055049e71f39f3ae3","valueType":"string"}
error: desde error
